File: Logistics.Web/ClientApp/src/app/models/item-category.ts
import { Item } from './item';
import { ItemGroupCategory } from './item-group-category';
import { Manufacturer } from './manufacturer';

export class ItemCategory {
  id: number;
  itemGroupCategoryId: number;
  label: string;
  isDeleted: boolean;
  itemGroupCategory: ItemGroupCategory;
  items: Item[];
  manufacturers: Manufacturer[];
}

File: Logistics.Data/ItemCategory.cs
using System.Collections.Generic;

namespace Logistics.Data
{
    public class ItemCategory
    {
        public int Id { get; set; }
        public int ItemGroupCategoryId { get; set; }
        public string Label { get; set; }
        public bool IsDeleted { get; set; }

        public ItemGroupCategory ItemGroupCategory { get; set; }

        public List<Item> Items { get; set; }
        public List<Manufacturer> Manufacturers { get; set; }
    }
}
